rabbitmq/assets/configuration/spec.yaml

```diff
@@ -29,6 +29,8 @@ files:
           - name: include_aggregated_endpoint
             description: |
               Whether or not to include metrics from the aggregated endpoint (/metrics).
+
+              Note: This option should be set to true when 'unaggregated_endpoint' is unspecified.
```

Comment:
small nit. if I'm understanding this correctly then I think it would be clearer to use `need to` instead of `should` here. I guess should to me still sounds like a choice, but given it breaks if aggregated is set to false and unaggegated is unspecified.

Reply:
Agreed, I'll use stronger language here.